src/v3/src/util/languageUtils.ts

```diff
@@ -21,6 +24,9 @@ export const loadLanguage = async (widgetProps: WidgetProps): Promise<void> => {
   const languageCode = getLanguageCode(widgetProps);
   const supportedLanguages = getSupportedLanguages(widgetProps);
 
+  // Odyssey language codes use '_' instead of '-' (e.g. zh-CN -> zh_CN)
+  const odyLanguageCode: string = languageCode.replace('-', '_');
```

Comment:
Reuse `i18next` from Odyssey

Reply:
Odyssey's i18next usage is pretty minimal AFAIK. It's only there for a few hardcoded strings, e.g., "Show/Hide password" for the visibility toggle aria attr